CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing to Chef Provisioning AWS
+
+## Resource/Provider Acceptance Criteria
+
+All resources and providers in Chef Provisioning AWS must fulfill the [common acceptance criteria]
+(https://github.com/chef/chef-provisioning/blob/master/CONTRIBUTING.md) for
+all Chef Provisioning projects.  In addition, Chef Provisioning AWS has the following additional
+acceptance criteria:
+
+1.  When an AWS object supports dependent objects, the resource must also have a `:purge` action which destroys all dependent AWS objects, as well as the specified object.
+  1. This is primarily for development purposes, and should be documented as such.
+  1. The action must print a warning when called, so that users know that the resource does not properly clean up data bag references to purged objects.
+1. If an object can be tagged in the Amazon Ruby SDK, the resource must have an `aws_tags` attribute.
```

Comment:
If we're not going to specify that they have to use our tagging module, we should specify the parameters for their `aws_tags` attributes (probably Hash, but should be explicit).